README.md

```diff
@@ -1217,6 +1217,56 @@ const MyElement = (props, chren) => <div>Hello, {chren()}!</div>;
 
 
 
+#### `vdom.elements (...elements)`
+
+Returns an array of functions that create elements.
+
+```js
+const [ div, p ] = skate.vdom.elements('div', 'p');
+div(p.bind('test'));
```

Comment:
I like that you suggested that. I've thought about that in the past but never did it because you have to return a function, but just realised that essentially that's what we have to do anyways by passing a function that it executes as children.

Though, it's not clear how would siblings work then because you'd currently have to do:

```js
div(() => {
  p('test');
  p('test');
});
```

Maybe:

```js
div(p('test'), p('test'));
```

But for performance, wouldn't it be better to have a single function like in the first example? I like the syntax, I'm just not sure yet if the perf implications are worth it.

If we did do this, then we could implement https://github.com/skatejs/skatejs/issues/751 and then just do:

```js
render() {
  return div(p('test'));
}
```

Instead of having to call `vdom.element()`.